ArduCopter/takeoff.cpp

```diff
@@ -90,7 +90,7 @@ void Mode::_TakeOff::do_pilot_takeoff(float& pilot_climb_rate_cm)
         copter.attitude_control->set_throttle_out(throttle, true, 0.0);
         // tell position controller to reset alt target and reset I terms
         copter.pos_control->init_z_controller();
-        if (throttle >= 0.9 || 
+        if (throttle >= MIN(copter.g2.takeoff_throttle_max, 0.9) || 
```

Comment:
no need to worry about the user putting in 0 or negative values?

Reply:
No, this will result in the old take-off behaviour.